pybliometrics/scopus/utils/startup.py

```diff
@@ -48,18 +49,28 @@ def check_sections(config: Type[ConfigParser]) -> None:
         if not config.has_section(section):
             raise NoSectionError(section)
 
+
 def check_default_paths(config: Type[ConfigParser], config_path: Type[Path]) -> None:
     """Auxiliary function to check if default cache paths exist.
-    If not, the paths are writen in the config"""
+    If not, the paths are writen in the config.
+    """
     for api, path in DEFAULT_PATHS.items():
         if not config.has_option('Directories', api):
             config.set('Directories', api, str(path))
             with open(config_path, 'w', encoding='utf-8') as ouf:
                 config.write(ouf)
+
+
+def create_cache_folders(config: Type[ConfigParser]) -> None:
+    """Auxiliary function to create cache folders."""
+    section = 'Directories'
+    for api in config.options(section):
+        path = Path(config.get(section, api))
         for view in VIEWS[api]:
             view_path = path/view
             view_path.mkdir(parents=True, exist_ok=True)
```

Comment:
Looking at https://docs.python.org/3.11/library/configparser.html#mapping-protocol-access, I think we can improve this further.
Could you please try the last bullet point, i.e., `parser.items(section, raw, vars)`? Also, we don't need a variable to store the section name.

Reply:
Good idea!